.gitignore

```diff
@@ -53,6 +55,7 @@ playground.xcworkspace
 # Carthage
 #
 # Add this line if you want to avoid checking in source code from Carthage dependencies.
+Carthage/
```

Comment:
with this, we will still be able to keep everything in Carthage/iOS/Build?

Reply:
I have not worked with Carthage before - just had a bunch of modified files in my local repo after calling `swift run --package-path Tools/Bootstrap/ Bootstrap .`
Maybe "Carthage/Build" is enough to ignore - will update this PR.

Reply:
that's true what you said. The reason is some of the cached builds are invalid. I've just tried the command and it generate a lot of untracked files. I think it is ok to ignore `Carthage/Build` then and we will force add it in case there is any update.